### ISSUE TITLE
Accessibility Tab redundant

### ISSUE THREAD

**Issue by ykeara** [OpenVR-Advanced-Settings/OpenVR-AdvancedSettings#27]
As it stands, the accessibility tab appears to be completely redundant with playspace tab, as Such should be removed.

**Comment by ykeara** [comment]
It does appear it has a toggle feature to say snap height form + .5 to + 0 that is not built in elsewhere

**Comment by username223** [comment]
Since the `AccessibilityTabController` inherits from `PttController` it either has to be refactored together with the `AudioTabController` or deleted. Simply deleting it is way easier than trying to wrangle that mess into something useful only to then delete it.

It is being deleted as part of #51.

**Comment by username223** [comment]
Solved by PR #79.
